.github/scripts/mega-app-create-app.sh

```diff
@@ -51,16 +51,27 @@ if [[ "$FRAMEWORK" == 'vue' ]]; then
     fi
 fi
 
-if [[ "$BUILD_TOOL" == 'cli' && "$FRAMEWORK" == 'react-native' ]]; then
+if [[ "$FRAMEWORK" == 'react-native' ]]; then
     echo "rm -rf $MEGA_APP_NAME" # Remove $MEGA_APP_NAME if it exists
     rm -rf $MEGA_APP_NAME
-    echo "npx react-native@${BUILD_TOOL_VERSION} init $MEGA_APP_NAME --version $FRAMEWORK_VERSION"
-    npx react-native@${BUILD_TOOL_VERSION} init $MEGA_APP_NAME --version $FRAMEWORK_VERSION
-    # React-Native, since 0.71.8,
-    # no longer shows warning "npm WARN exec The following package was not found and will be installed: react-native@0.71.8",
-    # so we log the package.json to check the versions
-    echo "cd $MEGA_APP_NAME"
-    cd $MEGA_APP_NAME
-    echo "cat package.json"
-    cat package.json
+    if [[ "$BUILD_TOOL" == 'cli' ]]; then
+        echo "npx react-native@${BUILD_TOOL_VERSION} init $MEGA_APP_NAME --version $FRAMEWORK_VERSION"
+        npx react-native@${BUILD_TOOL_VERSION} init $MEGA_APP_NAME --version $FRAMEWORK_VERSION
+        # React-Native, since 0.71.8,
+        # no longer shows warning "npm WARN exec The following package was not found and will be installed: react-native@0.71.8",
+        # so we log the package.json to check the versions
+        echo "cd $MEGA_APP_NAME"
+        cd $MEGA_APP_NAME
+        echo "cat package.json"
+        cat package.json
```

Comment:
I realize this part already was there before, but can we do something like `yarn react-native info` instead?

Reply:
Done 9a4bf3b58ff8419dcdebd7abfca277c55122fcd8